### PR TITLE
[M1722b] use short sha for non-prod version numbers

### PR DESCRIPTION
**Pull request by saanobhaai**

### CODE REVIEW

**Comment by saanobhaai** [comment]
[new update version approach](https://trello.com/c/S0tJUxlp/1722-new-update-version-approach)

File: .github/workflows/_cdk.yml [hunk 1]
```diff
@@ -40,19 +40,23 @@ jobs:
...
+      - name: Set variables and version for prod
         if: startsWith(github.ref, 'refs/tags')
         run: |
           echo "TARGET=prod" >> "$GITHUB_ENV"
           echo "REACT_APP_AUTH0_CLIENT_ID=Yahf5mGpQDyYOqbvQlaKE1WqBnUeFPj0" >> "$GITHUB_ENV"
           echo "REACT_APP_AUTH0_AUDIENCE=https://api.datamermaid.org" >> "$GITHUB_ENV"
           echo "REACT_APP_MERMAID_API=https://api.datamermaid.org/v1" >> "$GITHUB_ENV"
           echo "REACT_APP_MERMAID_REFERENCE_LINK=https://public.datamermaid.org/mermaid_attributes.xlsx" >> "$GITHUB_ENV"
+          echo "export const versionNumber = '${{ github.ref_name }}'" > src/version.js
+          cat src/version.js
```
Comment (aspcanada) [inline .github/workflows/_cdk.yml:59]: what is the point of cat here?

File: .github/workflows/_cdk.yml [hunk 2]
```diff
@@ -40,19 +40,23 @@ jobs:
...
         run: yarn install --frozen-lockfile
 
-      - name: Create version file
+      - name: Create version file for dev
+        if: startsWith(github.ref, 'refs/heads/develop')
         run: |
-          echo "export const versionNumber = '${{ github.ref_name }}'" > src/version.js
+          short_sha=$(git rev-parse --short ${{ github.sha }})
+          echo "export const versionNumber = '$short_sha'" > src/version.js
           cat src/version.js
```
Comment (aspcanada) [inline .github/workflows/_cdk.yml:48]: what is the point of cat here?

**Review by aspcanada — APPROVED** [review]
otherwise, looks good

**Pull request merged**